### PR TITLE
virsh_migrate: Fix variable initialize issue

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py [hunk 1]
```diff
@@ -1296,7 +1298,10 @@ def check_migration_timeout_suspend(params):
                                  mount_dir=mount_dir,
                                  restore_selinux=local_selinux_bak)
     # cleanup pre migration setup for remote machine
-    migrate_setup.migrate_pre_setup(dest_uri, params, cleanup=True)
+    if migrate_setup:
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py]: Looks good to me, please consider L1126 too

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py [hunk 2]
```diff
@@ -747,6 +748,7 @@ def check_migration_timeout_suspend(params):
     asynch_migration = False
     ret_migrate = True
 
+
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py]: remove this empty line for travis failure.

Comment (dzhengfy) [reply libvirt/tests/src/virsh_cmd/domain/virsh_migrate.py]: @balamuruhans Thanks. Done

**Comment by waynesun09** [comment]
ACK

**Pull request merged**